ui/admin/tests/acceptance/host-catalogs/read-test.js

```diff
@@ -71,14 +74,33 @@ module('Acceptance | host-catalogs | read', function (hooks) {
       instances.hostCatalog.authorized_actions.filter(
         (item) => item !== 'read'
       );
-    await visit(urls.hostCatalog);
-    assert.notOk(find('main tbody .rose-table-header-cell:nth-child(1) a'));
+
+    await click(`[href="${urls.hostCatalogs}"]`);
+
+    assert.dom(`[href="${urls.hostCatalog}"]`).doesNotExist();
   });
 
   test('visiting an unknown host catalog displays 404 message', async function (assert) {
-    assert.expect(1);
+    assert.expect(2);
+    await click(`[href="${urls.hostCatalogs}"]`);
+    assert.dom(`[href="${urls.unknownHostCatalog}"]`).doesNotExist();
+
     await visit(urls.unknownHostCatalog);
     await a11yAudit();
-    assert.ok(find('.rose-message-subtitle').textContent.trim(), 'Error 404');
+
+    assert.strictEqual(
+      find('.rose-message-subtitle').textContent.trim(),
+      'Error 404'
+    );
```

Comment:
I think this also could have been: 

```suggestion
    assert.dom('.rose-message-subtitle').hasText('Error 404');
```